src/lib/get-property-id.js

```diff
@@ -3,11 +3,32 @@ import { getMerchantID } from '@paypal/sdk-client/src';
 
 import type {
   Config,
-  Container
+  Container,
+  ContainerSummary
 } from '../types';
 
-import { getPropertyId, setPropertyId } from './local-storage';
+import { getPropertyId, setPropertyId, setContainer, getValidContainer } from './local-storage';
 
+/* Takes the full container and transforms it into
+a format better suited for use by the SDK */
+const parseContainer = (container : Container) : ContainerSummary => {
+  const offerTag = container.tags.find(tag => tag.tag_definition_id === 'offers');
```

Comment:
Internet Explorer doesn't support `Array.prototype.find` :( 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Babel _should_ be transpiling polyfills for us. I'll try to do some manual testing on IE before next Wednesday.

![Screen Shot 2019-10-09 at 2 31 52 PM](https://user-images.githubusercontent.com/13542922/66522145-97dd2e80-eaa1-11e9-8aca-495558cd6b5f.png)

Reply:
Came across this in paypal-checkout-components:
![Screen Shot 2019-10-09 at 2 41 04 PM](https://user-images.githubusercontent.com/13542922/66522690-e0e1b280-eaa2-11e9-9cf6-b83a71a4a17e.png)

Reply:
Updated.